src/main/java/org/team1540/robot2023/RobotContainer.java

```diff
@@ -28,6 +32,16 @@ public class RobotContainer {
     // Commands
 
     public RobotContainer() {
+        boolean ENABLE_PNEUMATICS = false;
```

Comment:
Either make this a constant or make this a public static final field since we might want to access this elsewhere